dsa-solutions/lc-solutions/3000-3099/3005-count elements with maximum frequency.md

```diff
@@ -0,0 +1,152 @@
+---
+id: Count Elements With Maximum Frequency
```

Comment:
replace `id: Count Elements With Maximum Frequency` to `id: count-elements-with-maximum-frequency`